modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+    ├── _images -> ../_images
+    ├── modules -> ../modules
+    ├── assembly-kogito-configuring.adoc
+    ├── assembly-kogito-creating-running.adoc
+    ├── assembly-kogito-decision-engine.adoc
+    ├── assembly-kogito-developing-decision-services.adoc
+    ├── assembly-kogito-developing-process-services.adoc
+    ├── assembly-kogito-using-dmn.adoc
+    ├── assembly-kogito-using-drl.adoc
+    ├── ...
+├── modules
+    ├── configuration
+        ├── con-data-index-service.adoc
+        ├── con-jobs-service.adoc
+        ├── proc-data-index-service-enable-security.adoc
+        ├── ref-data-index-service-queries.adoc
+        ├── ...
+    ├── creating-running
+        ├── con-kogito-automation.adoc
+        ├── con-kogito-modelers.adoc
+        ├── proc-kogito-app-examples-running.adoc
+        ├── ref-kogito-app-examples.adoc
+        ├── ...
+    ├── decision-services  // Contains modules related to multiple assemblies.
+        ├── con-decision-tables.adoc
+        ├── con-dmn.adoc
+        ├── con-drl.adoc
+        ├── proc-dmn-model-creating.adoc
+        ├── ref-dmn-model-example.adoc
+        ├── ...
+    ├── process-services
+        ├── ...
+    ├── ...
+├── titles-enterprise
+    ├── kogito-configuring
+        ├── _images -> ../../_images
+        ├── assemblies -> ../../assemblies
+        ├── master.adoc
+        └── master-docinfo.xml
+    ├── kogito-creating-running
+        ├── ...
+    ├── kogito-developing-decision-services  // Contains multiple assemblies.
+        ├── ...
+    ├── ...
+├── pantheon2.yml
+└── README.md
+----
+
+This repository structure contains the following key components:
+
+* `_artifacts`: Directory with document attribute files and other metadata files. The underscore in the folder name is an optional indication of a non-content folder.
+* `_images`: Directory with image files, optionally divided into sub-directories by subject matter. The underscore in the folder name is an optional indication of a non-content folder.
+* `assemblies`: Directory with assembly files. You collect and compile assemblies into documents (titles) using the `titles-enterprise` folder.
+* `modules`: Directory with module files, optionally divided into sub-directories by subject matter.
+* `titles-enterprise`: Directory with folders for each enterprise document or _title_ to be built in Pantheon. Each document folder contains a `master.adoc` file where you include one or more relevant assemblies that are built as part of the document. The `master-docinfo.xml` file contains metadata that is required by `bccutil` (and `ccutil`) when converting to DocBook format. (This metadata file might be deprecated in a future release of `bccutil` and Pantheon 2.) In repositories that are single-sourced with a community project, a separate `titles-community` folder might be dedicated to community configurations and output, if applicable.
+* `pantheon2.yml`: YAML file with metadata that is required for publishing content on Pantheon 2:
```

Comment:
This is RH internal (unfortunately, despite promises to the contrary, Pantheon is not being developed as an open(-source) project. Therefore, references to the tool should be kept out of these guidelines, so as not to detract from their value for other organizations and upstream projects.

If we need internal RH guidelines for publishing, they should be kept separately from this content.